### PR TITLE
Add use of use \Civi\Test\Api3TestTrait to make testwriting easier

### PR DESCRIPTION
**Pull request by eileenmcnaughton**

### CODE REVIEW

**Comment by seamuslee001** [comment]
This looks good to me @totten

File: src/CRM/CivixBundle/Resources/views/Code/test-api.php.php [hunk 1]
```diff
@@ -12,6 +12,7 @@
  * @group headless
  */
 class <?php echo $testClassName ?> extends \PHPUnit_Framework_TestCase implements HeadlessInterface, HookInterface, TransactionalInterface {
+  use \Civi\Test\Api3TestTrait;
```
Comment (seamuslee001) [inline src/CRM/CivixBundle/Resources/views/Code/test-api.php.php]: @eileenmcnaughton should we include the DocTrait as well?

Comment (eileenmcnaughton) [reply src/CRM/CivixBundle/Resources/views/Code/test-api.php.php]: @seamuslee001 possibly - but this is like from July & I don't even remember it now so I don't want to scope creep it at the moment

**Comment by totten** [comment]
Thanks, @eileenmcnaughton  @seamuslee001. Agree this makes sense.

**Pull request merged**